cloudify_ansible/__init__.py

```diff
@@ -78,10 +118,37 @@ def wrapper(playbook_path=None,
             else:
                 sources = get_source_config_from_ctx(ctx)
 
+        # store sources in node runtime_properties
+        _get_instance(ctx).runtime_properties['sources'] = sources
+        _get_instance(ctx).update()
+
         try:
             create_playbook_workspace(ctx)
-            playbook_path = handle_site_yaml(
-                playbook_path, additional_playbook_files, ctx)
+            # check if source path is provided [full path/URL]
+            if playbook_source_path:
+                # here we will combine playbook_source_path with playbook_path
+                playbook_tmp_path = playbook_source_path
+                split = playbook_source_path.split('://')
+                schema = split[0]
+                if schema in ['http', 'https']:
```

Comment:
What if the user provides a URL that is not a zip archive? Or what if it is a tar?